spec/lib/myfinance/resources/payable_account_spec.rb

```diff
@@ -0,0 +1,85 @@
+require "spec_helper"
+
+describe Myfinance::Resources::PayableAccount do
+  describe "#create", vcr: true do
+    let(:params) { { due_date: '2015-08-15', amount: 150.99 } }
+    subject { client.payable_accounts.create(3798, params) }
+
+    context "with success" do
+      it "creates a new object" do
+        expect(subject.id).to eq(1215631)
+        expect(subject.due_date).to eq(Date.new(2015, 8, 15))
+        expect(subject.entity_id).to eq(3798)
+        expect(subject.status).to eq(1)
+        expect(subject.status_name).to eq('unpaid')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.